GUI/MainInstall.cs

```diff
@@ -96,6 +97,31 @@ public partial class Main
                 installCanceled = true;
             };
 
+            GUI.user.RaiseMessage("About to install...\r\n");
```

Comment:
Well, that sucks. I'll poke that when I get home and see if I can understand what's going on.

Reply:
Sorry, I meant "I haven't tested running a changeset that only contains updated to installed mods, but I suspect it would suffer the same problem"